modules/tinymce/src/themes/silver/main/ts/ui/general/Button.ts

```diff
@@ -138,12 +141,26 @@ export const renderButtonSpec = (
     tag: 'button',
     classes,
     attributes: {
-      title: translatedText // TODO: tooltips AP-213
+      title: translatedText
```

Comment:
I assume we want a TODO to remove this later?

Reply:
Added